eng/pipelines/smoke-tests.yml

```diff
@@ -0,0 +1,26 @@
+jobs:
+  - job: SmokeTest   
+    variables:
+      node_version: "10.16.0"
```

Comment:
We should use the variable template here: https://github.com/Azure/azure-sdk-for-js/blob/master/eng/pipelines/templates/variables/globals.yml

The value is in `NodeVersion` 

We can also specify the image to use the default os vm image
```
jobs: 
  - job: SmokeTest
    variables: 
      - template: ./templates/variables/globals.yml

    pool: 
      vmImage: $(OSVmImage)
    ...
```